source/Calamari.Tests/Calamari.Tests.csproj

```diff
@@ -212,7 +256,8 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>xcopy "$(SolutionDir)..\tools\Octodiff\Octodiff.exe" "$(TargetDir)" /y</PostBuildEvent>
+    <PostBuildEvent>xcopy "$(SolutionDir)..\tools\Octodiff\Octodiff.exe" "$(TargetDir)" /y
+xcopy "$(SolutionDir)packages\NuGet.CommandLine.2.8.3\tools\Nuget.exe" "$(TargetDir)" /y</PostBuildEvent>
```

Comment:
One of the tests themselves directly reference this to pre-package a set of files for "deployment".  The existing tests do this but by referencing back into the packages directly rather than including it as a build dependency.
 suppose we could replace all this with the dlls directly.